README.md

```diff
@@ -42,10 +42,22 @@ make k8s=1.29
 # build an AMI with a specific Kubernetes version and a specific OS distro
 make k8s=1.29 os_distro=al2023
 
+# build Neuron AMI with a specific Kubernetes version
+make k8s=1.29 os_distro=al2023 accelerator_vendor=neuron
+
+# build Nvidia AMI with a specific Kubernetes version and default driver major version
+make k8s=1.29 os_distro=al2023 accelerator_vendor=nvidia
+
+# build Nvidia AMI with a specific Kubernetes version and a specific driver major version
+make k8s=1.29 os_distro=al2023 accelerator_vendor=nvidia nvidia_major_driver_version=555
```

Comment:
I don't think we need this in the top-level README, it shouldn't be commonly used. Lets just document it in the template docs.

Reply:
okay